index.html

```diff
@@ -0,0 +1,444 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width= , initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="./style/main.css">
+    <title>KMarketer</title>
+</head>
+<body>
+    <div class="content">
+        <div class="header header--bg-gray">
+            <div class="header__item">
+                <h1 class="logo">KMarketer</h1>
+                <ul class="navigation">
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">Home</a>
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">Product</a>
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">Pricing</a>
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">Contact</a>
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">
+                            <img src="./assets/svgs/search.svg" alt="">
+                        </a>
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">
+                            <img src="./assets/svgs/cat.svg" alt="">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="header__content">
+                <h1 class="site-header site-header--large " >
+                    Experts are here solve 
+                    <br/>
+                    your business problem.
+                </h1>
+                <p class="header__sub-content">
+                    We know how large objects will act, but things on a 
+                    <br/>small scale just do not act that way.
+                </p>
+                <div class="header__action">
+                    <button class="button button--primary button--rounded-full" >
+                        Get Quote Now
+                    </button>
+                    <button class="button button--secondary-outline button--rounded-full" >
+                        Learn More
+                    </button>
+                </div>
+            </div>
+        </div>
+    
+        <div class="container container--bg-white">
+            <h1 class="site-header site-header--medium site-header--center" >
```

Comment:
This is a block, and am re-using it in every section if I make it an element I don't think that would be Okay because  I will be copying all styles which are the same over and over. I think I can change the name maybe